translations/messages.fr.xlf

```diff
@@ -0,0 +1,3095 @@
+        <target>La partie que vous avez essayé de rejoindre n&apos;existe plus.</target>
+      </trans-unit>
+      <trans-unit id="2009811124619716606" datatype="html">
+        <source>Create an online game</source>
+        <target>Créer une partie en ligne</target>
+      </trans-unit>
+      <trans-unit id="5561648955936795459" datatype="html">
+        <source>Use simple mechanics to push 6 of the opponent&apos;s pieces out of the board!</source>
+        <target>Utilisez des mécaniques simples pour pousser 6 pièces adverses hors du plateau !</target>
+      </trans-unit>
+      <trans-unit id="6262000022886850348" datatype="html">
+        <source>The international version of the famous African strategy game!</source>
+        <target>La version internationale du fameux jeu de stratégie africain !</target>
+      </trans-unit>
+      <trans-unit id="1207528295664437538" datatype="html">
+        <source>Stack your pieces and control as many stacks as you can to win!</source>
+        <target>Empilez vos pièces pour en contrôler un maximum et gagner !</target>
+      </trans-unit>
+      <trans-unit id="7930050431770016664" datatype="html">
+        <source>An enhanced tic-tac-toe where piece can encapsule other and prevent them to win.</source>
+        <target>Un morpion amélioré où les pièces peuvent en encapsuler d'autres pour éviter la défaite.</target>
+      </trans-unit>
+      <trans-unit id="8971165322320863634" datatype="html">
+        <source>An antiquity-war inspired game. Be the first to pierce your opponent&apos;s lines!</source>
+        <target>Un jeu inspiré de l'antiquité. Soyez le premier à percer les lignes adverses !</target>
+      </trans-unit>
+      <trans-unit id="1787395418772268592" datatype="html">
+        <source>A hexagonal game of alignment. Insert your pieces on the board to capture your opponent&apos;s pieces!</source>
+        <target>Un jeu hexagonal d'alignement. Insérez vos pièces sur le plateau pour capturer les pièces de l'adversaire !</target>
+      </trans-unit>
+      <trans-unit id="6676975125770922470" datatype="html">
+        <source>The oldest strategy game still practiced widely. A territory control game.</source>
+        <target>Le plus vieux jeu de stratégie encore joué. Un jeu de contrôle de territoire</target>
+      </trans-unit>
+      <trans-unit id="8165475229121998889" datatype="html">
+        <source>Regroup your pieces to win. But your possible moves will often change!</source>
+        <target>Regroupez vos pièces pour gagner. Mais les mouvements possibles changent constamment !</target>
+      </trans-unit>
+      <trans-unit id="287142221400627248" datatype="html">
+        <source>The classical 4 in a row game!</source>
+        <target>Le classique Puissance 4 !</target>
+      </trans-unit>
+      <trans-unit id="6377255794894182998" datatype="html">
+        <source>Drop a piece, then rotate the board. The first player to align 5 pieces wins!</source>
```

Comment:
Oui en fait en anglais aussi ça c'est pas correct, comment on traduirait un quadrant, "rotate a bloc" ?

Reply:
rotate a block / rotate part of the board ?

Reply:
https://en.wikipedia.org/wiki/Pentago Quadrant is used there, perfect!

Reply:
bien trouvé, je vais aussi adapter dans le tuto !